pepper-apis/src/main/java/org/broadinstitute/ddp/json/Profile.java

```diff
@@ -50,6 +50,7 @@ public Profile(UserProfile other) {
         this.preferredLanguage = other.getPreferredLangCode();
         this.firstName = other.getFirstName();
         this.lastName = other.getLastName();
+        // NOTE: timezone is currently not exposed in Profile API JSON.
```

Comment:
I think we need to prioritize adding this feature. It is not a safe assumption that people will always be signing up from their preferred timezone- due to travel, or a host of reasons, especially given the prevalence of mobile devices- and we're going to need a way to update it.

Reply:
Updating user's timezone is a topic that came up. However, I don't think we fully understand the implications yet (e.g. what is the UX? what are the APIs? when and were can user update? etc). I suppose that needs input from rest of the team. I'll add a ticket for now.

Reply:
https://broadinstitute.atlassian.net/browse/DDP-4760

Reply:
Sounds good- I'll toss some comments into the ticket. That said, we should be more focused on the developer experience than the user one. The user never interacts directly with the API, the client software and developers do.